api/services/CaveService.js

```diff
@@ -150,14 +150,39 @@ module.exports = {
     }
   },
 
+  /**
+   * Get organizations that explored the cave
+   * @param {number} caveId
+   * @returns {Promise<Array>} Array of organizations with id and name
+   */
+  getExploringOrganizations: async (caveId) => {
+    const query = `
+      SELECT g.*
+      FROM t_grotto g
+             JOIN j_grotto_cave_explorer j ON g.id = j.id_grotto
+      WHERE j.id_cave = $1
+        AND g.is_deleted = false
+    `;
+    const result = await CommonService.query(query, [caveId]);
+    const grottos = result.rows;
+
+    if (!grottos || grottos.length === 0) {
+      return [];
+    }
+
+    await NameService.setNames(grottos, 'grotto');
+    return grottos;
+  },
+
   async getPopulatedCave(caveId, subEntitiesWhere = {}) {
     const cave = await TCave.findOne(caveId)
       .populate('author')
       .populate('reviewer')
       .populate('names')
       .populate('descriptions')
       .populate('entrances')
-      .populate('documents');
+      .populate('documents')
+      .populate('exploringGrottos');
```

Comment:
`.populate('exploringGrottos');`  is not needed when using `getExploringOrganizations(cave.id)` below